core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.spec.ts

```diff
@@ -2829,6 +2846,59 @@ describe('EditEmaEditorComponent', () => {
                         expect(spectator.query(byTestId(testId))).not.toBeNull();
                     });
                 });
+
+                describe('script and styles injection', () => {
+                    let iframe: Document;
```

Comment:
rename this var to `document`

Reply:
I'll set `iframeDocument` to be more precise.